alts/worker/runners/base.py

```diff
@@ -327,6 +327,35 @@ def install_package(self, package_name: str, package_version: str = None,
         return local['ansible-playbook'].run(
             args=cmd_args, retcode=None, cwd=self._work_dir)
 
+    @command_decorator(UninstallPackageError, 'uninstall_package',
+                       'Cannot uninstall package')
+    def uninstall_package(self, package_name: str, package_version: str = None,
+                        module_name: str = None, module_stream: str = None,
+                        module_version: str = None):
+        if package_name in CONFIG.uninstall_excluded_pkgs:
+            return
+        if package_version:
+            if self.pkg_manager in ('yum', 'dnf'):
+                full_pkg_name = f'{package_name}-{package_version}'
+            else:
+                full_pkg_name = f'{package_name}={package_version}'
+        else:
+            full_pkg_name = package_name
+
+        self._logger.info(f'Uninstalling {full_pkg_name} from {self.env_name}...')
```

Comment:
module `logging` uses `%-format string`, not critical
https://docs.python.org/3/howto/logging.html#logging-variable-data